src/reducers/authenticationReducer.js

```diff
@@ -50,6 +90,9 @@ export const DEFAULT_USER_SETTINGS = {
 export const DEFAULT_USER = {
   geolocation: Map(),
   settings: { ...DEFAULT_USER_SETTINGS },
+  // redux doesn't seem to detect updates with sets
```

Comment:
That's weird. 🤔

Reply:
I really tried but nothing seemed to update.. maybe we try later.